controllers/cars.js

```diff
@@ -38,18 +38,21 @@ exports.getCars = (req, res, next) => {
 };
 
 exports.getCar = (req, res, next) => {
-  const car = Car.findById(req.params.id);
-
-  const data = {
-    status: 200,
-    data: car
-  };
-  if (car != null) {
-    res.status(200).json(data);
-  } else {
-    data.status = 404;
-    res.status(404).json(data);
-  }
+  let result = Car.findById(req.params.id);
+
+  result
+    .then(car => {
+      let response =
+        car.rows.length === 0
+          ? new Response(200, car.rows, null, "Car not Found").response()
```

Comment:
Not found should be a `200` replace it with a right status code.

Reply:
Change updated in the next PR